tokio-console/src/state/tasks.rs

```diff
@@ -217,16 +218,38 @@ impl TasksState {
                     warnings: Vec::new(),
                     location,
                 };
-                task.lint(linters);
+                let recheck = task.lint(linters);
+                if recheck {
+                    self.pending_lint.insert(task.id);
```

Comment:
Instead add these to another `HashSet` that we won't check until next update (part of **PendingLintLogic**, see below).
```suggestion
                    next_pending_lint.insert(task.id);
```